libs/payments/stripe/src/lib/stripe.client.ts

```diff
@@ -166,6 +166,28 @@ export class StripeClient {
     return result as StripeResponse<StripeInvoice>;
   }
 
+  async invoicesUpdate(invoiceId: string, params?: Stripe.InvoiceUpdateParams) {
+    const result = await this.stripe.invoices.update(invoiceId, {
+      ...params,
+      expand: undefined,
+    });
+    return result as StripeResponse<StripeInvoice>;
+  }
+
+  async invoicesPay(invoiceId: string) {
+    try {
+      return await this.stripe.invoices.pay(invoiceId, {
```

Comment:
Can we eliminate the return value here so that the return signature is consistent?

Reply:
yes, its not being used so there's no issue with that